### PR TITLE
Express.js server for a Sign Protocol API tailored for Mt-Dog

### PR DESCRIPTION
**Pull request by amardeepio**

### CODE REVIEW

File: services/sign-protocol-interactor/dist/app.js
Comment (vishakh) [inline services/sign-protocol-interactor/dist/app.js:1]: Typically the `dist` folder is also added to `.gitignore`, no?

File: services/sign-protocol-interactor/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (vishakh) [inline services/sign-protocol-interactor/package.json:13]: No license please.

File: services/sign-protocol-interactor/dist/app.js
Comment (amardeepio) [reply services/sign-protocol-interactor/dist/app.js:1]: removed dist folder.

File: services/sign-protocol-interactor/package.json (hunk 1 above)
Comment (amardeepio) [reply services/sign-protocol-interactor/package.json]: done.

**Review by vishakh — APPROVED** [review]

**Pull request merged**